dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/bootstrap.properties

```diff
@@ -8,5 +8,4 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-bootstrap.include=../testports.properties
-com.ibm.ws.logging.trace.specification=*=info:concurrent=all:concurrencyPolicy=all:context=all
```

Comment:
Was the trace accidentally removed?  Having it enabled shouldn't impact the ability to run the TCK.

Reply:
Never mind. I see where it was converted to server.xml later.